app/models/QRCode.server.js

```diff
@@ -18,22 +18,16 @@ export async function getQRCodes(shop, graphql) {
     orderBy: { id: "desc" },
   });
 
-  if (!qrCodes.length) {
-    return qrCodes;
-  }
```

Comment:
This early-exit felt redundant. The mapping below takes care of it.

Reply:
It's a tricky one. I like early returns, but I had to read it a couple of times because it wasn't clear to me why it was returning `qrCodes` specifically if it was empty (or `length` was otherwise falsey).

In general, if I'm writing in a functional style, doing some fetching/filtering/mapping/etc, I don't do anything with an empty array, because the functional methods handle that fine.

@surma @developit could one of you tie-break this one?

Reply:
I prefer the explicit early return, but not the way it was written. I’d write

```js
if(qrCodes.length <= 0) return [];
```

Reply:
Yeah, I'd go with `=== 0`, because it removes the "when the fuck would this be less than 0??" question. And yes, I prefer the explicit return of the empty array. Sold.